CoreData/NSManagedObject+S2MAdditions.m

```diff
@@ -0,0 +1,306 @@
+//
+//  NSManagedObject+S2MAdditions.m
+//  S2MToolbox
+//
+//  Created by ParkSanggeon on 13. 10. 22..
+//  Copyright (c) 2013년 SinnerSchrader Mobile. All rights reserved.
+//
+
+#import "NSManagedObject+S2MAdditions.h"
+
+@implementation NSManagedObject (S2MAdditions)
+
++ (NSEntityDescription*)entityInManagedObjectContext:(NSManagedObjectContext*)moc_
+{
+#warning use mogenerator
+    return nil;
+}
+
+- (NSMutableDictionary *)propertyDictionary
```

Comment:
Should this be named 'attributesDictionary'? This method is only using attributes to create the dictionary, not relationships. Properties include attributes & relationships.